lib/filtered_rack_logger.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Works just like the regular Rails Rack request logger, but silences all requests to the
+# configured silenced request paths.
+#
+#   config.middleware.use(
+#     FilteredRackLogger,
+#     silenced: %w(/rails/active_storage)
+#   )
+class FilteredRackLogger < Rails::Rack::Logger
+  def initialize(app, silenced:)
+    super(app)
+    @silenced = silenced
+  end
+
+  def call(env)
+    if @silenced.any? { |path_prefix| env['PATH_INFO'].start_with?(path_prefix) }
+      Rails.logger.silence { @app.call(env) }
```

Comment:
Would it make sense to limit what kinds of requests get silenced? I'm worried about not being notified if something was actually wrong or broken.

Reply:
We're only silencing requests to `/rails/active_storage*` in development. These paths are Rails internals and only serve uploaded files. So if anything related images or audio is broken in development you can turn the regular logging middleware back on and see what's happening.